eravm-stable-interface/src/lib.rs

```diff
@@ -0,0 +1,136 @@
+use u256::{H160, U256};
+
+pub trait StateInterface {
```

Comment:
Dumb question: Is all this functionality required for tracers we want to implement initially? AFAIU, e.g. the call tracer is "readonly" (doesn't change the VM state). What are the examples of tracers mutating the VM state for the old VM? In general, since we don't need to have stable API right away, I'd suggest to have a minimal interface for the known uses cases (call tracer), and then iteratively add / improve on the interface as new tracers are developed.

Reply:
No, we definitely don't need all this.

However, I went over this design pretty carefully with Igor from the compiler team and this is the bare minimum of what they need.